src/utils/common.js

```diff
@@ -262,51 +268,57 @@ export function getEthereumProvider(network: string) {
   // Creating a provider to automatically fallback onto Etherscan
   // if INFURA is down
   return new providers.FallbackProvider([infuraProvider, etherscanProvider]);
-}
+};
+
+export const padWithZeroes = (value: string, length: number): string => {
+  let myString = value;
 
-export function padWithZeroes(value: string, length: number): string {
-  let myString = `${value}`;
   while (myString.length < length) {
     myString = `0${myString}`;
   }
+
   return myString;
-}
+};
 
-export function concatSig({ v, r, s }): string {
+export const concatSig = ({ v, r, s }): string => {
   const rSig = ethUtil.fromSigned(r);
   const sSig = ethUtil.fromSigned(s);
   const vSig = ethUtil.bufferToInt(v);
   const rStr = padWithZeroes(ethUtil.toUnsigned(rSig).toString('hex'), 64);
   const sStr = padWithZeroes(ethUtil.toUnsigned(sSig).toString('hex'), 64);
   const vStr = ethUtil.stripHexPrefix(ethUtil.intToHex(vSig));
+
   return ethUtil.addHexPrefix(rStr.concat(sStr, vStr)).toString('hex');
-}
+};
 
-export function ethSign(msgHex: String, privateKeyHex: string): string {
+export const ethSign = (msgHex: String, privateKeyHex: string): string => {
   const message = ethUtil.toBuffer(msgHex);
   const privateKey = ethUtil.toBuffer(privateKeyHex);
   const sigParams = ethUtil.ecsign(message, privateKey);
   const result = concatSig(sigParams);
+
   return result;
-}
+};
 
-export function getRandomString(): string {
+export const getRandomString = (): string => {
   return utils.bigNumberify(utils.randomBytes(32)).toHexString().slice(2);
-}
+};
 
-export function extractJwtPayload(jwtToken: string): Object {
+export const extractJwtPayload = (jwtToken: string): Object => {
   // extract: header (not needed), payload, signature (not needed)
   const [, encodedPayload] = jwtToken.split('.');
+
   // do not use Buffer.toJSON
   return JSON.parse(Buffer.from(encodedPayload, 'base64').toString());
```

Comment:
I think it's better to wrap this with try..catch and return `{}` if we fail to parse

Reply:
Updated.